### ISSUE TITLE
Need for some tips on setting parameters

### ISSUE THREAD

**Issue by saeedsltm** [AI4EPS/GaMMA#30]
Since there is no any document ready until now, It would be helpful to give us some tips and notes on how to select best parameters controling the final results. It seems the following are the most important ones that should be choosen with care while working in local or regional scales:
- dbscan_eps
- dbscan_min_samples
- max_sigma11
- any other?
Thanks.

**Comment by zhuwq0** [comment]
Hi, these parameters are explained in the README file.

**Comment by saeedsltm** [comment]
Yes, the definitions are provided, but how to select their optimized values? should we do it by  try and error?

**Comment by zhuwq0** [comment]
There are always some trade-offs: number of detections and false positive rate; speed and accuracy. This depends on your focus.

**Comment by saeedsltm** [comment]
I ran the code in one day continus data for the Turkey-Syria earthquakes (2023-02-06 7.8Mw & 7.5Mw) and it almost done. The P and S picks for small quakes are well idetified, detected and associated. The only problem is that the output catalog does not contain large events (like M> 5). 
Shoud we consider different range of parameters:
These are what i have used:
- dbscan_eps = 15
- dbscan_min_samples=3
- min_picks_per_eq=6
- max_sigma11=1.5

**Comment by zhuwq0** [comment]
Hi, first I would check how many picks available for these large events in the input data; secondly for debugging, I would make max_sigma11 very large to remove filtering.

**Comment by saeedsltm** [comment]
Hi, following your suggestion i increased max_sigma11 to 2.5s. Now a strange situation has occured!:
- Choosing a narrow time window ( 15 min) for the whole processing (picking + association) results in dtecting more events in which previously missed.
- Choosing a broader time window ( 60 min) for the whole processing (picking + association) causes once again some events to be missed (not accosiated).

Do you have any idea? I also increased dbscan_eps to 20, but still missing some large events while working with 60 min time window.
